src/Generator.Bind/CSharpSpecWriter.cs

```diff
@@ -695,13 +695,45 @@ private string GetDeclarationString(Parameter p, bool override_unsafe_setting, S
         {
             StringBuilder sb = new StringBuilder();
 
+            List<string> attributes = new List<string>();
             if (p.Flow == FlowDirection.Out)
             {
-                sb.Append("[OutAttribute] ");
+                attributes.Add("OutAttribute");
             }
             else if (p.Flow == FlowDirection.Undefined)
             {
-                sb.Append("[InAttribute, OutAttribute] ");
+                attributes.Add("InAttribute");
+                attributes.Add("OutAttribute");
+            }
+
+            if (!String.IsNullOrEmpty(p.ComputeSize))
+            {
+                int count;
+                if(Int32.TryParse(p.ComputeSize, out count))
+                {
+                    attributes.Add(String.Format("CountAttribute(Count = {0})", count));
+                }
+                else
+                {
+                    if(p.ComputeSize.StartsWith("COMPSIZE"))
+                    {
+                        //remove the compsize hint, just keep comma delimited param names
+                        var len = "COMPSIZE(".Length;
+                        var computed = p.ComputeSize.Substring(len, (p.ComputeSize.Length - len) - 1);
+                        attributes.Add(String.Format("CountAttribute(Computed = \"{0}\")", computed));
+                    }
+                    else
+                    {
+                        attributes.Add(String.Format("CountAttribute(Parameter = \"{0}\")", p.ComputeSize));
+                    }
+                }
+            }
+
+            if(attributes.Count != 0)
+            {
+                sb.Append("[");
+		sb.Append(string.Join(", ", attributes));
```

Comment:
Good eyes! Must of been when I did a quick tweak in vim on my Windows machine which isn't set up correctly yet.